vsts/pipelines/templates/_integrationJobTemplate.yml

```diff
@@ -179,43 +174,11 @@ jobs:
         - template: _setReleaseTag.yml
         - template: _buildTemplate.yml
 
-- job: Job_PhpIntegrationTests8_0
-  displayName: Run Php Integration Tests 8.0
-  dependsOn:
-    - Job_BuildImage_Latest
-    - Job_BuildImage_LtsVersions
-    - Job_BuildImage_Jamstack
-    - Job_BuildImage_GithubActions
-    - Job_BuildImage_VsoFocal
-    - Job_BuildImage_Full
-    - Job_BuildImage_Cli
-    - Job_BuildImage_Buildpack
-    - Job_RuntimeImages
-  pool:
-    name: AzurePipelines-EO
-    demands:
-      - ImageOverride -equals AzurePipelinesUbuntu20.04compliant
-  variables:
-    skipComponentGovernanceDetection: true
-  timeoutInMinutes: 300
-  steps:
-  - script: |
-      echo "##vso[task.setvariable variable=BuildBuildImages;]false"
-      echo "##vso[task.setvariable variable=BuildRuntimeImages;]false"
-      echo "##vso[task.setvariable variable=TestBuildImages;]false"
-      echo "##vso[task.setvariable variable=TestRuntimeImages;]false"
-      echo "##vso[task.setvariable variable=TestIntegrationCaseFilter;]category=php-8.0"
-      echo "##vso[task.setvariable variable=TestIntegration;]true"
-      echo "##vso[task.setvariable variable=PushBuildImages;]false"
-      echo "##vso[task.setvariable variable=PushRuntimeImages;]false"
-      echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]false"
-    displayName: 'Set variables'
-  - template: _setReleaseTag.yml
-  - template: _buildTemplate.yml
-
-- job: Job_PhpIntegrationTests7_4
-  displayName: Run Php Integration Tests 7.4
-  dependsOn:
+# Php integration tests
+- ${{ each phpVersion in parameters.phpVersions }}:
+  - job:
+    displayName: 'Run Php ${{ phpVersions }} Integration Tests'
```

Comment:
Looks like the nightly pipeline can't be execute since this is `phpVersions` instead of `phpVersion`